source/faq.txt

```diff
@@ -113,33 +113,33 @@ is on the other side of the planet and has a latency of 10000
 milliseconds, setting the ``connectTimeoutMS`` to anything lower will
 prevent the driver from ever connecting to that member.
 
-Should I Use "socketTimeoutMS" as a Way of Preventing Long-Running Operations From Slowing Down the Server?
--------------------------------------------------------------------------------------------------------------
-
-No, you should **not** use ``socketTimeoutMS`` to end operations that
-may run for too long and slow down the application. Attempting to do so
-may not achieve the intended result.
-
-Closing the socket causes a reconnect of the driver's connection pool,
-introducing latency to any other queued up operations. Chronically slow
-operations will, therefore, cause a large number of reconnect requests,
-negatively impacting throughput and performance.
-
-Also, closing the socket does not terminate the operation; it will
-continue to run on the MongoDB server, which could cause data
-inconsistencies if the application retries the operation on failure.
-
-However, there are important use cases for ``socketTimeoutMS`` -
-consider the following cases:
-
-- A MongoDB process erroring out
-- A misconfigured firewall causing a socket connection without sending a ``FIN`` packet
-
-In those cases, there is no way to detect that the connection has died.
-Setting the ``socketTimeoutMS`` is essential to ensure that the sockets
-are closed correctly. A good rule of thumb is to set ``socketTimeoutMS``
-to two to three times the length of the slowest operation that runs
-through the driver.
+What Happens to Running Operations If the Client Disconnects?
+-------------------------------------------------------------
+
+Starting in {+mdb-server+} version 4.2, the server terminates
+certain operations such as aggregations and find operations if the
+client disconnects the operations successfully completes. To see a full
+list of operations affected by this behavior, see the :manual:`Server version
+4.2 release notes </release-notes/4.2/#client-disconnection>` in the
+Server manual.
+
+Other operations, such as write operations, continue to run on the
+{+mdb-server+} even if the client disconnects. This behavior can cause data
```

Comment:
Not changing because it seems like the mechanism is not that the server "allows" certain operations to continue running, but rather that it marks certain operations for termination.